browser/assets/css/style2.scss

```diff
@@ -0,0 +1,74 @@
+---
+permalink: /assets/css/style2.css
+---
+
+@import 'jekyll-theme-cayman';
+
+html {
+	height: 100%;
+}
+
+body {
+	margin: 0;
+	height: 100%;
+}
+
+a {
+	color: #058b8c
+}
+
+#container {
+	height: 100%;
+}
+
+#sidebar {
+	display: inline-block;
+	vertical-align: top;
+	height: 100%;
+	width: 25%;
```

Comment:
Is this a bit wide?  Maybe 20% and shrink the sidebar font a bit?

Reply:
Will give that a try.